qrenderdoc/Windows/TextureViewer.ui

```diff
@@ -158,6 +161,9 @@
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
+      <property name="toolTip">
+       <string>Auto Fit</string>
```

Comment:
Similarly, this can be a better explanation than just a name:

> Automatically fit the black and white points of the range to the minimum and maximum values contained in the texture.